apps/web/src/main.tsx

```diff
@@ -19,33 +19,57 @@ import { Layout } from "./components/layout.tsx";
 import Login from "./components/login/index.tsx";
 import { ErrorBoundary, useError } from "./ErrorBoundary.tsx";
 import { trackException } from "./hooks/analytics.ts";
-
+import UIFallbackSpinner from "./components/common/UIFallbackSpinner.tsx";
+
+type LazyComp = Promise<{
+  default: React.ComponentType;
+}>;
+const wrapWithUILoadingFallback = (
+  lazyComp: LazyComp,
+): LazyComp =>
+  lazyComp.then((mod) => ({
+    // deno-lint-ignore no-explicit-any
+    default: (p: ComponentProps<any>) => (
+      <UIFallbackSpinner>
```

Comment:
eu botaria a implementacao do UIFallbackSpinner aqui para evitar ficar criando arquivos

Reply:
Sounds good, will do it, good catch